src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +121,27 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        //
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {
+            var portsToIgnore = new List<int>();
+            foreach(var item in serviceNameSpace.Items) {

```

Comment:
💡 Thoughts: Can we make this `for each` simpler for maintainability of when this funct will get added with more in future.

There is nested `foreach loop` in this can we `single responsibility` so one inner iteration `IgnorePorts()` loops through and make it replaceable.